libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -201,6 +219,12 @@ class BreezSDKMapper {
             "message": bitcoinAddressData.message,
         ]
     }
+
+    static func dictionaryOf(checkMessageResponse: CheckMessageResponse) -> [String: Any?] {
```

Comment:
`checkMessageResponse.isValid` should always return a value so the return type can be `[String: Any]`